splunk-otel-android/src/main/java/com/splunk/rum/NoOpSplunkRum.java

```diff
@@ -79,4 +79,9 @@ public void integrateWithBrowserRum(WebView webView) {
     void flushSpans() {
         // no-op
     }
+
+    @Override
+    public void experimentalSetScreenName(String screenName) {
+        // no-op
+    }
```

Comment:
There is an overload for this that also needs no-op coverage.

```suggestion
    @Override
    public void experimentalSetScreenName(String screenName) {
        // no-op
    }
    
    @Override
    public void experimentalSetScreenName(String screenName, String spanType) {
      // no-op
    }
```